Makefile

```diff
@@ -15,6 +15,8 @@ migrate:
 upgrade-deps:
 	pip-compile --rebuild --header --index --annotate  requirements.in
 	pip-compile --rebuild --header --index --annotate  requirements-dev.in
+	sed -i 's/^-e //g' requirements.txt
```

Comment:
I'd like a comment referencing the issue above the 2 sed lines. It's fine in the commit message, but in such a case, more is better.